src/frontend/packages/ui/types/index.ts

```diff
@@ -1,7 +1,15 @@
+export interface CourseData {
+  school_name: string;
+  course_name: string;
+  course_run?: string;
```

Comment:
This is LMS-specific (edx mostly). Maybe we should be more generic using:

```suggestion
  platform: string;
  organization?: string;
  course_name: string;
  course_run?: string;
```
WDYT?

Reply:
Agree, It would ask a bit of data manipulation, but make the code more generic

Reply:
I have renamed it in the back-end, WDYT?

Reply:
Looks great! We need to keep in mind multi-tenancy (_i.e._ multiple platforms support).